packages/ozone/src/mod-service/status.ts

```diff
@@ -203,6 +203,24 @@ const getSubjectStatusForRecordEvent = ({
   return {}
 }
 
+export const moderationSubjectStatusQueryBuilder = (db: DatabaseSchema) => {
+  return db
+    .selectFrom('moderation_subject_status as mss')
+    .selectAll('mss')
```

Comment:
I do get the appeal of the shortened versions of these table names, but they leak out into the calling functions and I find it hard to balance the mental state of them. Generally I try to only use abbreviated names like this when the variable is confined to some scoped function & the context is contained within a dozen or two lines. Even though it's wordier, I'm inclined to just use the table names for these.